pyscitt/pyscitt/cli/main.py

```diff
@@ -7,7 +7,6 @@
     create_did_web,
```

Comment:
Please update the `pyscitt` CLI version

Reply:
I want to add another feature that helps to preview the embedded receipt and extract the payload before bumping up the version, is that all right with you?